xml/ay_bigfile.xml

```diff
@@ -17165,44 +17161,65 @@ mv /var/run/zypp.sav /var/run/zypp.pid
   <title>Advanced Linuxrc Options</title>
   <info/>
   <para>
-   Linuxrc is a program used for setting up the kernel for installation
-   purposes. It allows the user to load modules, start an installed system,
-   a rescue system or an installation via &yast;.
+   <command>Linuxrc</command> is a small program that runs after the kernel loads, but before &ay;
+   or other stages. It prepares the system for installation. It allows the user
+   to load modules, start an installed system or a rescue system, and to guide
+   the operation of &yast;.
   </para>
+  <note>
+   <title>Linuxrc Settings Are Not The Same As &ay; Settings</title>
+   <para>
+    Some <command>linuxrc</command> settings have the same names as settings
+    used by &ay; in the <filename>autoyast.xml</filename> file. Beware that this
```

Comment:
Would be more concise (and contain the same information) if you shorten it to: "This does not mean...."